app/routes/funding/what-funding.js

```diff
@@ -24,35 +27,37 @@ module.exports = [{
   options: {
     validate: {
       payload: Joi.object({
-        standard: Joi.array().items(Joi.string()).single()
+        standard: Joi.required()
       }),
       failAction: async (request, h, error) => {
         const { agreement } = await cache.get(request)
         const { funding } = agreement
-        const eligibleFunding = await getFunding(request)
+        const eligibleFunding = await getFunding(request) ?? undefined
         if (!eligibleFunding) {
           return h.view('no-response').takeover()
         }
         return h.view('funding/what-funding', new ViewModel(eligibleFunding, funding, error)).code(400).takeover()
       }
     },
     handler: async (request, h) => {
-      const { standard } = request.payload
+      const standards = typeof (request.payload.standard) === 'string' ? [request.payload.standard] : request.payload.standard
       const { data, agreement } = await cache.get(request)
 
-      const funding = data?.eligibleFunding.filter(x => standard.includes(x.code)) ?? []
+      const eligibleFunding = (data?.eligibleFunding ?? []).filter(fundingObj => standards.includes(fundingObj.code))
```

Comment:
Variables funding renamed to eligibleFunding as there are 2 variables within the cache with these names and this line refers to the latter.